ffv1.md

```diff
@@ -26,9 +26,11 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 ## Definitions
 
 -------- --------------------------------------------------------------
-`Sample`: The smallest addressable representation of a color component or a luma component in a frame. Examples of sample are Luma, Blue Chrominance, Red Chrominance, Alpha, Red, Green, Blue.
+`Container`: Format that encapsulates `Frames` and (when required) a `Configuration Record` into a bitstream.
 
-`Pixel`: The smallest addressable representation of a color in a frame. It is composed of 1 or more samples.
+`Sample`: The smallest addressable representation of a color component or a luma component in a `Frame`. Examples of sample are Luma, Blue Chrominance, Red Chrominance, Alpha, Red, Green, Blue.
```

Comment:
I suggest a more puristic naming: Examples of sample are Luma, Blue minus Luma, Red minus Luma, Alpha, […]

Reply:
I prefer the term as Jerome has it. I think the terms "blue minus luma / red minus luma" are misleading by oversimplifying a transform as the color space would likely add weights and constants. As examples "Blue Chrominance, Red Chrominance" would apply more broadly.